### PR TITLE
refactor: reduce PR checklist to something useful

### PR DESCRIPTION
**Pull request by alessandro-mariotti-zupit**
## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] Does the code follow the team's coding standards and best practices?
- [ ] Is the code well-organized and easy to understand?
- [ ] Are file, variable and function names clear and meaningful?
- [ ] Are there any duplicated code blocks that can be refactored into reusable functions?

## Functionality
- [ ] Does the PR implement the intended functionality or fix the issue described in the title and description?
- [ ] Are there any edge cases that have been overlooked?

## Testing
- [ ] Are there sufficient tests covering the changes?
- [ ] Are there any missing test cases that need to be added?
- [ ] Has the code been tested in different environments if applicable (e.g., browsers, platforms, devices)?

## Documentation
- [ ] Are the changes well-documented and easy to understand?
- [ ] Are there any missing or outdated comments in the code?
- [ ] Is the PR title clear and detailed enough to provide necessary context?

## Performance
- [ ] Do the changes have any impact on the system's performance?
- [ ] Are there any potential bottlenecks that need to be addressed?

## Dependencies
- [ ] Have the changes introduced any new dependencies or updated existing ones?
- [ ] Are the dependencies up-to-date and secure?

## Compatibility
- [ ] Will the changes have any adverse effects on other parts of the system?
- [ ] Do they work well with other existing features?

## Continuous Integration/Deployment (CI/CD)
- [ ] Have the changes passed the CI/CD pipeline?
- [ ] Are there any deployment considerations or steps that need to be taken into account?

## Code Review language
- [ ] Is the feedback provided in a constructive and respectful manner?
- [ ] Has the reviewer avoided personal attacks or harsh language?
- [ ] Are there any suggestions for improvement or alternative approaches?

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,41 +1,7 @@
 ## Checklist before requesting a review
 - [ ] I have performed a self-review of my code
 - [ ] Does the code follow the team's coding standards and best practices?
-- [ ] Is the code well-organized and easy to understand?
-- [ ] Are file, variable and function names clear and meaningful?
 - [ ] Are there any duplicated code blocks that can be refactored into reusable functions?
-
-## Functionality
-- [ ] Does the PR implement the intended functionality or fix the issue described in the title and description?
```
Comment (pietro-devigili-zupit) [inline .github/pull_request_template.md:9]: Io terrei anche questa riga, almeno come casistica mi capita "spesso" di vedere cose extra dentro una PR che mi portano a farla spezzare in due. Ti riporto l'ultimo caso come esempio

_Per semplificare il lavoro a chi fa le pull request, cerca di tenere separate le modifiche operative (relative alla issue) da quelle che vanno a modificare files che non riguardano la issue per altri motivi (es. csharpier)
Anche per il futuro, se dovessimo andare a vedere cosa è stato modificato per introdurre la funzionalità troveremmo tanto rumore inutile_

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,41 +1,7 @@
...
-
-## Testing
-- [ ] Are there sufficient tests covering the changes?
-- [ ] Are there any missing test cases that need to be added?
-- [ ] Has the code been tested in different environments if applicable (e.g., browsers, platforms, devices)?
-
-## Documentation
-- [ ] Are the changes well-documented and easy to understand?
-- [ ] Are there any missing or outdated comments in the code?
+- [ ] Are there tests on the new code?
```
Comment (alessandro-mariotti-zupit) [inline .github/pull_request_template.md:5]:
```suggestion
- [ ] Are there tests on the new code?
- [ ] Does the PR implement the intended functionality or fix the issue described in the title and description?
```

**Review by pietro-devigili-zupit — APPROVED** [review]

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,41 +1,8 @@
...
-- [ ] Do the changes have any impact on the system's performance?
-- [ ] Are there any potential bottlenecks that need to be addressed?
-
-## Dependencies
-- [ ] Have the changes introduced any new dependencies or updated existing ones?
-- [ ] Are the dependencies up-to-date and secure?
-
-## Compatibility
-- [ ] Will the changes have any adverse effects on other parts of the system?
-- [ ] Do they work well with other existing features?
```
Comment (omar-muscatello-zupit) [inline .github/pull_request_template.md:32]: I would include at least one of these questions. This is something we need to think about and may not always be at the forefront of developers' minds.

**Review by omar-muscatello-zupit — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,41 +1,8 @@
...
-
-## Continuous Integration/Deployment (CI/CD)
 - [ ] Have the changes passed the CI/CD pipeline?
```
Comment (alessandro-mariotti-zupit) [inline .github/pull_request_template.md:8]:
```suggestion
- [ ] Have the changes passed the CI/CD pipeline?
- [ ] Will the changes have any adverse effects on other parts of the system and do they work well with other existing features?
```

**Review by omar-muscatello-zupit — APPROVED** [review]
